scipy/stats/tests/test_sampling.py

```diff
@@ -312,6 +312,100 @@ def check_discr_samples(rng, pv, mv_ex):
     assert pval > 0.1
 
 
+@pytest.mark.parametrize('method', ["NumericalInverseHermite", "NumericalInversePolynomial"])
+class TestQRVS:
+    def test_qrvs(self, method):
```

Comment:
This looks the same, although a more descriptive/unique name (distinct from `test_QRVS`) below would be good. 

It looks like you cut out only parts related to QMC and left the rest behind, which seems right.

Reply:
I kept the name of the class and tried to improve the names of the methods